### ISSUE TITLE
Added message close button ("X") has no accessible name.

### ISSUE THREAD

**Issue by derekjackson-das** [danny-avila/LibreChat#3641]
**Issue Description**: When another conversation thread is added to the chat, the button to close the added conversation does not have an accessible name and it is not clear what it does.

**WCAG Criteria**: [4.1.2 Name, Role, Value - A](https://www.w3.org/WAI/WCAG21/Understanding/name-role-value)

**Snippet**: ```<button class="text-token-text-secondary flex-shrink-0" type="button" aria-label="Close added conversation">
    <svg xmlns="http://www.w3.org/2000/svg" width="24" height="24" fill="none" viewBox="0 0 24 24" class="icon-lg" aria-hidden="true">```

**Notes**:
Add an aria-label to the button to give it a name. The name should start with the word "close" because that is most likely the action associated with the icon, e.g., "Close added conversation".
Because the SVG is identified as an image, it should also be hidden from assistive technology. aria-hidden="true" is the simplest way to hide the image without affecting the button.

<img width="1761" alt="Screenshot 2024-08-14 at 4 38 44 PM" src="https://github.com/user-attachments/assets/7dcd536e-dbfc-4ee9-8f9d-7ad8a423f00f">
